taxi_service/settings.py

```diff
@@ -124,6 +124,12 @@
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
+
+]
```

Comment:
The 'STATICFILES_DIRS' setting specifies additional locations the staticfiles app will traverse to collect static files. Ensure that the 'static' directory exists at the specified path.